src/Facebook/Facebook.php

```diff
@@ -586,4 +589,72 @@ public function videoToUpload($pathToFile)
     {
         return new FacebookVideo($pathToFile);
     }
+
+    /**
+     * Upload video
+     *
+     * @param int $target
+     * @param string $pathToFile
+     * @param string $accessToken
+     * @param int $maxTransferTries
+     *
+     * @return array
+     *
+     * @throws FacebookSDKException
+     */
+    public function uploadVideo($target, $pathToFile, $accessToken, $maxTransferTries = 5)
```

Comment:
Graph API has the check for versions. And it will handle the logic differently based on the api version. User can set whatever version they want, but here, for this function, since we use chunked upload, if we want a value for $this->defaultGraphVersion, it should be at least v2.3.